crates/bevy_gltf/src/lib.rs

```diff
@@ -98,6 +98,8 @@ pub struct Gltf {
     /// Named animations loaded from the glTF file.
     #[cfg(feature = "bevy_animation")]
     pub named_animations: HashMap<String, Handle<AnimationClip>>,
+    /// The gltf root of the gltf asset, see <https://docs.rs/gltf/latest/gltf/struct.Gltf.html>. Only has a value when `GltfLoaderSettings::include_source` is true.
+    pub source: Option<gltf::Gltf>,
```

Comment:
I think this is ok. 
I  wondered if it would be better hidden behind a feature flag like  ```named_animations``` but it would not really make sense at the bevy level imho.

Reply:
Yeah, I think this makes sense, the idea is that if you don't need the extra data for some assets you can save on memory for those models specifically. Sometimes we only need the model data we're extracting normally.